test/integration/merkle-streamer/ll/constructor/constructor.t.sol

```diff
@@ -80,5 +90,88 @@ contract Constructor_MerkleStreamerLL_Integration_Test is MerkleStreamer_Integra
         vars.actualAllowance = dai.allowance(address(constructedStreamerLL), address(lockupLinear));
         vars.expectedAllowance = MAX_UINT256;
         assertEq(vars.actualAllowance, vars.expectedAllowance, "allowance");
+
+        // Blast configuration
+        vars.actualBlastYieldMode = YieldMode(blastMock.readYieldConfiguration(address(constructedStreamerLL)));
+        vars.expectedBlastYieldMode = YieldMode.VOID;
+        assertEq(uint8(vars.actualBlastYieldMode), uint8(vars.expectedBlastYieldMode), "blastYieldMode");
+
+        (,,, vars.actualBlastGasMode) = blastMock.readGasParams(address(constructedStreamerLL));
+        vars.expectedBlastGasMode = GasMode.VOID;
+        assertEq(uint8(vars.actualBlastGasMode), uint8(vars.expectedBlastGasMode), "blastGasMode");
+
+        vars.actualBlastGovernor = blastMock.governorMap(address(constructedStreamerLL));
+        vars.expectedBlastGovernor = address(0);
+        assertEq(vars.actualBlastGovernor, vars.expectedBlastGovernor, "blastGovernor");
+    }
+
+    function test_ConstructorWhen_RebasingAsset() external {
```

Comment:
@andreivladbrg

1. Should I create another internal function for the repeat code between `test_ConstructorWhen_RebasingAsset` and `test_Constructor`?
OR
2. Should I remove the repeat code from `test_ConstructorWhen_RebasingAsset` and only keep the last 4 Blast related assertions?
OR
3. Is it fine this way?

Reply:
I don't understand why we need these constructor tests 😅

We **are not** interacting with these contracts at creation, so we don't need to do anything.

It is like testing `ERC20` functions.

Reply:
Yes, you are correct. We don't need them. I wrote it when we were configuring yield in the Lockup constructor which we don't do anymore.